### PR TITLE
Add and updating the feedback point

### PR DESCRIPTION
**Pull request by sriLatha2328**

### CODE REVIEW

File: lib/screens/oneOnOne/update_1on1_view.dart [hunk 1]
```diff
@@ -513,9 +517,22 @@ class _UpdateOneoneOneViewState extends State<UpdateOneoneOneView> {
             Visibility(
               visible: hasAccessForUpdate1on1,
               child: TextButton(
-                  onPressed: () {
-                    _displayTextInputDialog(
-                        true, false, "Yet to Improve point", context);
+                  onPressed: () async {
+                    final result = await Navigator.push(
```
Comment (NagarajuKamatham) [inline lib/screens/oneOnOne/update_1on1_view.dart]: @sriLatha2328 you can move this code to common function.

File: lib/screens/oneOnOne/update_1on1_view.dart [hunk 2]
```diff
@@ -666,9 +696,23 @@ class _UpdateOneoneOneViewState extends State<UpdateOneoneOneView> {
             Visibility(
               visible: hasAccessForUpdate1on1,
               child: TextButton(
-                  onPressed: () {
-                    _displayTextInputDialog(
-                        true, true, "Good at point", context);
+                  onPressed: () async {
+                    // Navigate to Screen B and wait for the result
```
Comment (NagarajuKamatham) [inline lib/screens/oneOnOne/update_1on1_view.dart]: @sriLatha2328 same write common function

File: lib/screens/oneOnOne/update_1on1_view.dart [hunk 3]
```diff
@@ -850,7 +906,7 @@ class _UpdateOneoneOneViewState extends State<UpdateOneoneOneView> {
 
   refreshGoodAtList() {
     setState(() {
-     // buildGoodAtList();
+      // buildGoodAtList();
```
Comment (NagarajuKamatham) [inline lib/screens/oneOnOne/update_1on1_view.dart]: @sriLatha2328 remove commented code

**Review by NagarajuKamatham — CHANGES_REQUESTED** [review]

**Review by NagarajuKamatham — APPROVED** [review]

File: lib/screens/oneOnOne/AddOrUpdate_1on1_feedback_points.dart [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+  Widget build(BuildContext context) {
+    var isFromGoodAtList = widget.isFromGoodAtList;
+    return Scaffold(
+      appBar: AppBar(
+        leading: IconButton(
+          icon: const Icon(Icons.arrow_back, color: Colors.black),
+          onPressed: () {
+            if (enteredText.isNotEmpty) {
+              showValidationAlert(
+                  context, "Good at/Yet to Improve points will not be saved");
```
Comment (janakirampalepu) [inline lib/screens/oneOnOne/AddOrUpdate_1on1_feedback_points.dart]: @sriLatha2328  can you use constants for these.

CC : @NagarajuKamatham

File: lib/screens/oneOnOne/AddOrUpdate_1on1_feedback_points.dart [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+            } else {
+              Navigator.pop(context);
+            }
+          },
+        ),
+        backgroundColor: Colors.white,
+        title:  Text(
+          isFromGoodAtList ? "Good At" : "Yet to Improve",
```
Comment (janakirampalepu) [inline lib/screens/oneOnOne/AddOrUpdate_1on1_feedback_points.dart]: @sriLatha2328  can you use constants for these.

CC : @NagarajuKamatham

File: lib/screens/oneOnOne/AddOrUpdate_1on1_feedback_points.dart [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+              });
+            }),
+      ),
+    );
+  }
+
+  showValidationAlert(BuildContext context, String alertText) {
+    // set up the button
+    Widget okButton = TextButton(
+      child: const Text("OK"),
```
Comment (janakirampalepu) [inline lib/screens/oneOnOne/AddOrUpdate_1on1_feedback_points.dart]: @sriLatha2328  can you use constants for these.

CC : @NagarajuKamatham

**Review by janakirampalepu — CHANGES_REQUESTED** [review]

**Review by janakirampalepu — APPROVED** [review]

**Pull request merged**